test/test.js

```diff
@@ -681,7 +681,7 @@ describe('sanitizeHtml', function() {
   it('should not crash due to tag names that are properties of the universal Object prototype', function() {
     assert.equal(
       sanitizeHtml('!<__proto__>!'),
-      '!!');
+      '!&lt;__proto__&gt;!');
```

Comment:
@boutell Do you have thoughts about this resolution? It's still not crashing, as is the goal of the test, but the output is different. I'm not totally clear on why the tag is being escaped, regardless.

@TrySound Do you know why this is being escaped instead of discarded?

Reply:
I don't remember to be honest. Will check

Reply:
@abea Alright, I figured it out. Previously `<__proto__>` was parsed as html tag. In this commit valid characters for html element start were restricted to a-zA-Z. So here proto "tag" is escaped as text.
https://github.com/fb55/htmlparser2/pull/497

Reply:
According to this, escaping is a sensible response:

https://stackoverflow.com/questions/22066705/invalid-tags-in-html-abc-vs-1234

I don't think we're introducing a bc break here because tags not starting with a letter are not compliant anyway, and even in earlier versions of HTML they got treated as text, not tags. It's OK.

Reply:
However a bigger issue is that we allow options to be passed directly to htmlparser2 when calling sanitize-html.

So moving to 6.x is a bc break requiring a major version bump of our own — if that option set has changed in a non-backwards-compatible way. If it has just been extended in a backwards compatible way it's fine, but the htmlparser2 docs have to be checked on this point before acceptance.

Reply:
From what I see here https://github.com/fb55/htmlparser2/releases/tag/v5.0.0 only decodeEntities is changed but it was set by this project anyway.

v6 changes also do no break sanitize-html https://github.com/fb55/htmlparser2/releases/tag/v6.0.0